anyway/flask_app.py

```diff
@@ -1939,3 +1948,100 @@ def get_roles_list() -> Response:
         status=HTTPStatus.OK,
         mimetype="application/json",
     )
+
+
+@app.route("/user/get_organization_list", methods=["GET"])
+@roles_accepted(BE_CONST.Roles2Names.Admins.value)
+def get_organization_list() -> Response:
+    orgs_list = db.session.query(Organization).all()
+    send_list = []
+    for org in orgs_list:
+        send_list.append(org.name)
+
+    return app.response_class(
+        response=json.dumps(send_list),
+        status=HTTPStatus.OK,
+        mimetype="application/json",
+    )
+
+
+@app.route("/user/add_organization", methods=["POST"])
+@roles_accepted(BE_CONST.Roles2Names.Admins.value)
+def add_organization() -> Response:
+    reg_dict = request.json
+
+    if reg_dict is None:
+        return return_json_error(Es.BR_BAD_JSON)
+
+    name = reg_dict.get("name")
+    if not name:
+        return return_json_error(Es.BR_FIELD_MISSING)
+
+    org = db.session.query(Organization).filter(Organization.name == name).first()
+    if not org:
+        org = Organization(name=name, create_date=datetime.datetime.now())
+        db.session.add(org)
+        db.session.commit()
+    return Response(status=HTTPStatus.OK)
+
+
+@app.route("/user/remove_user_from_org", methods=["POST"])
+@roles_accepted(BE_CONST.Roles2Names.Admins.value)
+def remove_user_from_org() -> Response:
+    reg_dict = request.json
+    if reg_dict is None:
+        return return_json_error(Es.BR_BAD_JSON)
+
+    email = reg_dict.get("email")
+    if not email:
+        return return_json_error(Es.BR_NO_EMAIL)
+    user = get_user_by_email(db, email)
+    if user is None:
+        return return_json_error(Es.BR_USER_NOT_FOUND, email)
+
+    org = reg_dict.get("org")
+    if not org:
+        return return_json_error(Es.BR_FIELD_MISSING)
+    org_obj = db.session.query(Organization).filter(Organization.name == org).first()
```

Comment:
better to use .one() instead of .first() because there should be only one matching org